blocks/folder-blocks/dashboard/index.tsx

```diff
@@ -177,7 +177,10 @@ export default function (props: FolderBlockProps) {
                 <BlockComponent
                   {...props}
```

Comment:
maybe mark `{...props}` as soon to be deprecated also so we don't forget. but eh it doesn't really matter